unimol/pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "unimol"
+version = "1.0.1"
+description = "Tools for Uni-Mol."
+license = "MIT"
+readme = "README.md"
+authors = [{ name = "DP Technology", email = "unimol@dp.tech" }]
+requires-python = ">=3.10,<4.0"
+dependencies = [
+    "numpy >=2.1.1",
+    "pandas >=2.2.3",
+    "scikit-learn-extra >=0.3.0",
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+]
```

Comment:
Do we want to change any of this info  like the author emails?

Reply:
I guess we can add in some additional authors for this fork sure

Reply:
i dont mind either way about adding new others, just figured it was kinda odd to see their author email